deploy/Jenkinsfile

```diff
@@ -121,9 +121,9 @@ pipeline {
     }
 
     // Configured via the UI to allow concurrent PR builds
-    triggers {
-        cron('H H(0-7) * * *')
-    }
+    // triggers {
+    //     cron('H H(0-7) * * *')
+    // }
```

Comment:
commented but not removed?

Reply:
For future reference. Ideally, all configuration should be done via this file, but of course our setup is not made for that.